smartsim/_core/control/manifest.py

```diff
@@ -155,14 +153,15 @@ def __str__(self) -> str:
                     output += f"Parameters: \n{fmt_dict(model.params)}\n"
             output += "\n"
 
-        if self.db:
-            output += db_header
-            output += f"Shards: {self.db.num_shards}\n"
-            output += f"Port: {str(self.db.ports[0])}\n"
-            output += f"Network: {self.db._interfaces}\n"
-            output += f"Batch Launch: {self.db.batch}\n"
-            if self.db.batch:
-                output += f"{str(self.db.batch_settings)}\n"
+        if self.dbs:
```

Comment:
It should be possible to remove this `if`, as `dbs` cannot be `None` (and the loop will not iterate over an empty list).